### PR TITLE
Update Changelog for 2.5-alpha

### PR DESCRIPTION
**Pull request by daschuer**

### CODE REVIEW

**Comment by JoergAtGithub** [comment]
https://github.com/mixxxdj/mixxx/pull/4637 and https://github.com/mixxxdj/mixxx/pull/11035 were backported to 2.4 (see #11264). Please remove them from the 2.5 changelog therefore.

**Comment by daschuer** [comment]
Done and up to date.

File: CHANGELOG.md [hunk 1]
```diff
@@ -14,16 +13,23 @@
 * Playlist feature: add 'Shuffle playlist' sidebar action [#12498](https://github.com/mixxxdj/mixxx/pull/12498)
 * New built-in effect: Glitch [#11329](https://github.com/mixxxdj/mixxx/pull/11329)
 * Fullscreen toggle rework [#11566](https://github.com/mixxxdj/mixxx/pull/11566)
+* Playlists: Uupdate of playlist labels after adding tracks [#12866](https://github.com/mixxxdj/mixxx/pull/12866) [#12761](https://github.com/mixxxdj/mixxx/issues/12761)
```
Comment (JoergAtGithub) [inline CHANGELOG.md:16]:
```suggestion
* Playlists: Update of playlist labels after adding tracks [#12866](https://github.com/mixxxdj/mixxx/pull/12866) [#12761](https://github.com/mixxxdj/mixxx/issues/12761)
```

File: CHANGELOG.md [hunk 2]
```diff
@@ -14,16 +13,23 @@
...
+* Tracks: custom text color for played tracks (qss) [#12744](https://github.com/mixxxdj/mixxx/pull/12744) [#5911](https://github.com/mixxxdj/mixxx/issues/5911) [#12912](https://github.com/mixxxdj/mixxx/pull/12912)
```
Comment (JoergAtGithub) [inline CHANGELOG.md:17]:
```suggestion
* Tracks: Custom text color for played tracks (qss) [#12744](https://github.com/mixxxdj/mixxx/pull/12744) [#5911](https://github.com/mixxxdj/mixxx/issues/5911) [#12912](https://github.com/mixxxdj/mixxx/pull/12912)
```

File: CHANGELOG.md [hunk 3]
```diff
@@ -14,16 +13,23 @@
...
+* History: show track count and duration in sidebar [#12811](https://github.com/mixxxdj/mixxx/pull/12811) [#12788](https://github.com/mixxxdj/mixxx/issues/12788)
```
Comment (JoergAtGithub) [inline CHANGELOG.md:18]:
```suggestion
* History: Show track count and duration in sidebar [#12811](https://github.com/mixxxdj/mixxx/pull/12811) [#12788](https://github.com/mixxxdj/mixxx/issues/12788)
```

File: CHANGELOG.md [hunk 4]
```diff
@@ -14,16 +13,23 @@
...
+* fixes around cratetablemodel, remove tracks + don't allow pasting tracks into locked playlists/crates or History [#12926](https://github.com/mixxxdj/mixxx/pull/12926)
+* Tootips: Improve `rate_up/down` tooltips, pitch vs. speed [#12590](https://github.com/mixxxdj/mixxx/pull/12590)
+* Shortkeys for track list management [#12020](https://github.com/mixxxdj/mixxx/pull/12020)
+* Track menu: rephrase "Reset" to "Clear" [#12955](https://github.com/mixxxdj/mixxx/pull/12955)
```
Comment (JoergAtGithub) [inline CHANGELOG.md:22]:
```suggestion
* Track menu: Rephrase "Reset" to "Clear" [#12955](https://github.com/mixxxdj/mixxx/pull/12955)
```

File: CHANGELOG.md [hunk 5]
```diff
@@ -111,10 +134,21 @@
 * fix/History: remove obsolete placeholder playlists [#12494](https://github.com/mixxxdj/mixxx/pull/12494)
 * Add missing Taglib dependency [#12830](https://github.com/mixxxdj/mixxx/pull/12830)
 * fix: typo ;) [#12726](https://github.com/mixxxdj/mixxx/pull/12726)
+* refactor: avoid temporary qlist allocation on midi sysex receive [#12843](https://github.com/mixxxdj/mixxx/pull/12843)
```
Comment (JoergAtGithub) [inline CHANGELOG.md:137]:
```suggestion
* refactor: Avoid temporary qlist allocation on midi sysex receive [#12843](https://github.com/mixxxdj/mixxx/pull/12843)
```

File: CHANGELOG.md [hunk 6]
```diff
@@ -111,10 +134,21 @@
...
+* SoundSourceMP3: fix unused function warning [#12847](https://github.com/mixxxdj/mixxx/pull/12847)
+* update libkeyfinder to 2.2.8 [#12853](https://github.com/mixxxdj/mixxx/pull/12853)
+* feat(ci): add eslint rule prefer-template [#12889](https://github.com/mixxxdj/mixxx/pull/12889)
```
Comment (JoergAtGithub) [inline CHANGELOG.md:140]:
```suggestion
* feat(ci): Add eslint rule prefer-template [#12889](https://github.com/mixxxdj/mixxx/pull/12889)
```

File: CHANGELOG.md [hunk 7]
```diff
@@ -111,10 +134,21 @@
...
+* Labeler: Add `qml` to labeler config [#12911](https://github.com/mixxxdj/mixxx/pull/12911)
+* WTrackMenu: Add missing wcoverartlabel.h include [#12924](https://github.com/mixxxdj/mixxx/pull/12924)
+* Fix clazy complains and naming [#12935](https://github.com/mixxxdj/mixxx/pull/12935)
+* src/library: sort files into sub-directories [#12956](https://github.com/mixxxdj/mixxx/pull/12956)
```
Comment (JoergAtGithub) [inline CHANGELOG.md:144]:
```suggestion
* src/library: Sort files into sub-directories [#12956](https://github.com/mixxxdj/mixxx/pull/12956)
```

**Review by JoergAtGithub — CHANGES_REQUESTED** [review]

**Comment by daschuer** [comment]
Done

**Comment by JoergAtGithub** [comment]
LGTM! Thank you!

**Pull request merged**